src/components/Dashboard/PendingTxListItem/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement } from 'react'
+import { Text } from '@gnosis.pm/safe-react-components'
+import { TransactionSummary } from '@gnosis.pm/safe-react-gateway-sdk'
+import { Link } from 'react-router-dom'
+import ChevronRightIcon from '@material-ui/icons/ChevronRight'
+import styled from 'styled-components'
+
+import { useAssetInfo } from 'src/routes/safe/components/Transactions/TxList/hooks/useAssetInfo'
+import { useKnownAddress } from 'src/routes/safe/components/Transactions/TxList/hooks/useKnownAddress'
+import { useTransactionType } from 'src/routes/safe/components/Transactions/TxList/hooks/useTransactionType'
+import { getTxTo } from 'src/routes/safe/components/Transactions/TxList/utils'
+import { boldFont, grey400, primary200, sm } from 'src/theme/variables'
+import { isMultisigExecutionInfo } from 'src/logic/safe/store/models/types/gateway.d'
+import Spacer from 'src/components/Spacer'
+import { CustomIconText } from 'src/components/CustomIconText'
+import { TxInfo } from 'src/routes/safe/components/Transactions/TxList/TxCollapsed'
+import { EMPTY_DATA } from 'src/logic/wallets/ethTransactions'
+
+const TransactionToConfirm = styled(Link)`
+  min-width: 270px;
+  height: 40px;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 25px auto 2fr 1.5fr;
+  gap: 4px;
```

Comment:
I would suggest auto sizing the first and last column (since they are aligned to their sides) and setting the same width for the 2 columns in the middle (and auto sizing the second one in case its empt) i.e. `grid-template-columns: auto 1fr minmax(auto, 1fr) auto` and also increasing the gap to 16px.